cranelift/codegen/src/isa/x86/abi.rs

```diff
@@ -649,6 +649,11 @@ fn system_v_prologue_epilogue(func: &mut ir::Function, isa: &dyn TargetIsa) -> C
 
     // Add CSRs to function signature
     let reg_type = ir::Type::int(u16::from(pointer_width.bits())).unwrap();
+    if isa.pointer_bits() == 32 {
+        let sp_arg =
+            ir::AbiParam::special_reg(reg_type, ir::ArgumentPurpose::Normal, RU::rsp as RegUnit);
```

Comment:
`ArgumentPurpose::Normal` is not correct.

Reply:
I agree. What would be appropriate here, assuming `stack_load` indeed isn't, as discussed below?